addons/mail/models/mail_activity.py

```diff
@@ -389,6 +392,35 @@ def _compute_display_name(self):
             name = record.summary or record.activity_type_id.display_name
             record.display_name = name
 
+    # ------------------------------------------------------
+    # Notifications
+    # ------------------------------------------------------
+
+    def _notify_owner_store(self, store, notification_type="mail.record/update_activity"):
```

Comment:
Why passing a store? Callers typically create one just to pass it. We might want to do it inside this method instead, removing the need for an extra parameter and reducing the complexity.